api/cmd/portainer/main.go

```diff
@@ -492,6 +504,11 @@ func buildServer(flags *portainer.CLIFlags) portainer.Server {
 		}
 	}
 
+	err = overrideSettingsFromFlags(dataStore, flags)
+	if err != nil {
+		log.Fatalf("failed updating settings from flags: %v", err)
```

Comment:
To differenciate from log in L503
```suggestion
		log.Fatalf("failed overriding settings from flags: %v", err)
```